### DISCUSSION TITLE
What can cause expressionManager to come back null?

### DISCUSSION THREAD

**Discussion by aMidnightNova** [pixiv/three-vrm#1171] [Help]
The VRM I have been using is an older VRM0 one, but I cant export it as 1.0 if needed which I started to do to test some stuff. I noticed though if I export it as a 1.0 the expressionManager is null after loading the VRM.

What are some things I can look into as the cause of this ?

**Comment by 0b5vr** [comment]
How do you export your VRM1.0 models?

**Reply by aMidnightNova** [reply]
I was using UniVRM 108 in Unity 2020.3.33f1.
I also tried 2019.4.31f1 and UniVRM 99
